tavern/testutils/pytesthook.py

```diff
@@ -107,6 +108,12 @@ def pytest_addoption(parser):
     )
 
 
+def merge_dicts(x, y):
```

Comment:
Should use `tavern.util.dict_util.deep_dict_merge` instead of defining this function which does the same thing